### DISCUSSION TITLE
UI for creating and editing collection items

### DISCUSSION THREAD

**Discussion by jakob-fuchs** [directus/directus#4131] [Feature Requests]
It's not visually clear that save button and menu are connected.

![create_item](https://user-images.githubusercontent.com/1150242/108231594-340d5000-7142-11eb-8045-319e0f79fb5f.png)

Using a different UI pattern could make this clearer. This could be one option: https://material-ui.com/components/button-group/#split-button

![Screenshot from 2021-02-17 17-00-58](https://user-images.githubusercontent.com/1150242/108232169-c6155880-7142-11eb-974d-fe805d8c199a.png)

Additionally the save menu is hidden when editing items until a field value changes. Is there any harm in letting the user close the modal the way he is used to (by clicking the check mark button)? As it is right now the user needs to click the back button if he chose to open the editing form and did not make any changes.

**Comment by jakob-fuchs** [comment]
> Additionally the save menu is hidden when editing items until a field value changes

Letting the user save without making any changes is somewhat comparable to how archiving an item is handled in the collection detail pages. Archived items can be archived again.

**Reply by rijkvanzanten** [reply]
> Archived items can be archived again.

? Archived items can be _un_ -archived again, however you can't un-save an item

**Reply by jakob-fuchs** [reply]
This is what I mean @rijkvanzanten 

The archive button does not get disabled for items that are already archived. I don't know if an actual query gets performed or not. So why not make it work the same way for saving when editing collection items? 

https://user-images.githubusercontent.com/1150242/108237919-ada83c80-7148-11eb-8fbc-fad236d0d3cc.mp4

**Reply by rijkvanzanten** [reply]
That's what I'm saying. Archived items have an _unarchive_ button instead of an _archive_ button. It toggles back-and-forth

**Reply by jakob-fuchs** [reply]
That's not the case on the collection detail page.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
